lib/containers/runtime.pm

```diff
@@ -62,8 +62,9 @@ sub up {
     die 'image name or id is required' unless $image_name;
     my $mode   = $args{daemon} ? '-d'    : '-it';
     my $remote = $args{cmd}    ? 'sh -c' : '';
-    my $ret    = $self->_rt_script_run(sprintf qq(run --rm %s %s %s '%s'), $mode, $image_name, $remote, $args{cmd});
-    record_info "Remote run on $image_name", $args{cmd};
+    my $rmi    = $args{rmi}    ? ''      : '--rm';
```

Comment:
Isn't this vice versa? BTW Haven't there been a bit of discussion about the `daemon` key already ? :)

Reply:
I would personally prefer a more understandable name, e.g. `remove_image`

    my $rmi = $args{remove_image} ? '' : '--rm'
    
And also, include that parameter in the `head/cut` block, as the others.

Reply:
> I would personally prefer a more understandable name, e.g. `remove_image`
> 
> ```
> my $rmi = $args{remove_image} ? '' : '--rm'
> ```
> 
> And also, include that parameter in the `head/cut` block, as the others.

Or `keep_image`, but yes. The same just different :D

Reply:
or `keep_container` (see my long comment)

Reply:
yea. sorry for that. i got lazy. i am going to make it `keep_container`. i like it